src/components/fb/ExistingFidelityBond.jsx

```diff
@@ -28,7 +29,9 @@ const ExistingFidelityBond = ({ utxo }) => {
             <Sprite symbol="clock" width="18" height="18" className={styles.icon} />
             <div className="d-flex flex-column">
               <div className={styles.label}>Locked until</div>
-              <div className={styles.content}>{utxo.locktime}</div>
+              <div className={styles.content}>
+                {utxo.locktime} ({fb.time.humanReadableDuration({ to: new Date(utxo.locktime).getTime() })}
```

Comment:
```suggestion
                {utxo.locktime} ({fb.time.humanReadableDuration({ to: new Date(utxo.locktime).getTime() })})
```

Missing closing bracket.

Reply:
Damn! 😅